<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trangchu</class>
 <widget class="QMainWindow" name="trangchu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trang chủ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>551</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ffffff;&quot;&gt;QUẢN LÝ THƯ VIỆN MUA/THUÊ TRUYỆN NHÓM 8&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_qlt">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>QUẢN LÝ TRUYỆN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_qlkh">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>QUẢN LÝ KHÁCH HÀNG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_qltt">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>QUẢN LÝ THUÊ TRUYỆN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_qlbt">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>QUẢN LÝ BÁN TRUYỆN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_bcdt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>BÁO CÁO DOANH THU</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>601</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../anhnen/R.jpg</pixmap>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>btn_qlt</zorder>
   <zorder>btn_qlkh</zorder>
   <zorder>btn_qltt</zorder>
   <zorder>btn_qlbt</zorder>
   <zorder>btn_bcdt</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
